packages/carousel/src/StretchSlide/StretchSlide.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useRef } from 'react'
+import PropTypes from 'prop-types'
+
+const StretchSlide = ({ slideContents, height, setHeight }) => {
+  const ref = useRef(null)
+
+  useEffect(() => {
+    ref.current.clientHeight > height && setHeight(ref.current.clientHeight)
+  })
```

Comment:
I think you should add `height` to the dependency array for this `useEffect`

Reply:
Added `height` and `setHeight` to the dependency array